.gitignore

```diff
@@ -1,3 +1,4 @@
 *~
 *.pyc
-\#*
\ No newline at end of file
+\#*
+.vscode
```

Comment:
You might be interested in ignoring this in all projects on your machine. 
See `core.excludesfile` in https://git-scm.com/docs/gitignore/1.7.12

`git config --global core.excludesfile /path/to/global/.gitignore`

Reply:
Can do.